Contact/Block/Contact.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Gorilla\Contact\Block;
+
+class Contact extends \Magento\Framework\View\Element\Template
+{
+    /**
+     * @inheritdoc
+     */
+    public function _prepareLayout()
+    {
+        parent::_prepareLayout();
+        $this->pageConfig->getTitle()->set(__('Sign Up'));
+        return $this;
+    }
+
+    /**
+     * Returns action url for contact form
+     *
+     * @return string
+     */
+    public function getFormAction()
```

Comment:
Using block classes is no longer recommended best practice. A View Model should be defined and used instead.